bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -180,4 +180,10 @@
 		<label>Lightning Distance</label>
 		<state pattern="%.0f %unit%" readOnly="true"/>
 	</channel-type>
+
+	<channel-type id="free-stack-size">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
What kind of data is this - does is have a unit (like bytes), or is it expressed as a percentage or similar?

Reply:
Same doubt raised here: https://community.openhab.org/t/fine-offset-weather-station-binding-discussion/134167/269. IMHO it could look like bytes. If this is in fact the case, the item type should be `Number:DataAmount`.

Reply:
Good point, I think it is `bytes` and so we should name it like `free-stack-memory`, what is your oppinion?

Reply:
It seems the manifacturer primarily uses the term "heap" rather than "stack". Of those proposed three options, I would intuitively opt for `free-heap-size`. However, I don't have any strong opinions/preferences. 🙂 ChatGPT about size vs. memory:

> Both terms are used interchangeably, and either can be appropriate depending on the context. If you are discussing the remaining capacity in terms of bytes, kilobytes, megabytes, etc., "free size" might be more suitable. If you are focusing on the concept of memory availability in a more general sense, "free memory" might be preferred.
>
> For example:
>
> "The heap has 10 MB of free size."
> "There is 2 GB of free memory in the heap."

To me the most important part is to make it consistent (channel id, channel type id, label and description) and to fix the UoM type and unit.